### ISSUE TITLE
Confirm committed code and branches

### ISSUE THREAD

**Issue by AlexanderPico** [cytoscape/appstore#71]
Stage site = master
Dev site = dev

GSoC = wip

**Comment by AlexanderPico** [comment]
apps-stage.cytoscape.org = `master` 👍 
cytoscape-app-dev.ucsd.edu = `wip` 👎 : the dev site should be running the `dev` branch

Once `dev` has all changes in current `master` then we should be careful to make all future changes in `dev` first, then test, then mege to `master` for staging and release.  Make sense?

**Comment by sarthak-srivastava** [comment]
Yes. Thanks! I got a little confused between branch - server mapping
